docs/blog/newsletter-july-2019.rst

```diff
@@ -0,0 +1,115 @@
+.. post:: July 3, 2019
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2019
+#####################################
+
+Hi, WTD friends, and welcome to the July newsletter! It's Beth A here writing from sweltering Europe, where I've been learning great new words like *canicule* and *Hitzewelle* - aka heatwave in French and German.
+
+(Fun fact: *canicule* comes from the Latin *dies caniculares*, or "dog days" - the most uncomfortable part of summer, so named because they come after the `helical rising <https://en.wikipedia.org/wiki/Heliacal_rising>`_ of Sirius, the Dog Star. Well, I thought it was interesting.)
+
+Aside from fascinating facts about the weather, we do have some actual news. Firstly, `tickets are on sale for the Australia conference </conf/australia/2019/news/au-2019-welcome/>`__, and `the call for proposals is also open </conf/australia/2019/cfp/>`__! The CFP closes on 9 August, and the conference itself is 14-15 November in Melbourne.
```

Comment:
"we do have some actual news" -> "we have our usual exciting compendium of announcements and writeups of <appropriate_adjective> Slack conversations"
or something a little jazzier -- current version feels a bit limp